src/core/Akka/Util/Option.cs

```diff
@@ -50,7 +58,7 @@ public Option(T value)
         /// </summary>
         /// <param name="value">The object to convert</param>
         /// <returns>The result of the conversion.</returns>
-        public static implicit operator Option<T>(T value) => new Option<T>(value);
+        public static implicit operator Option<T>(T value) => Create(value);
```

Comment:
LGTM

Reply:
We should enable `nullable` in the future for this project - there's ways to do it in increments.